modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/PassThroughHttpSender.java

```diff
@@ -131,15 +129,12 @@ public void init(ConfigurationContext configurationContext,
         MBeanRegistrar.getInstance().registerMBean(view, "Transport",
                  "passthru-" + namePrefix.toLowerCase() + "-sender");
         
-        proxyConfig = new ProxyConfigBuilder().parse(transportOutDescription).build();
-        if (log.isInfoEnabled() && proxyConfig.getProxy() != null) {
-            log.info("HTTP Sender using Proxy " + proxyConfig.getProxy() + " bypassing " + 
-                proxyConfig.getProxyBypass());
-        }
-        
+        proxyConfig = new ProxyConfigBuilder().build(transportOutDescription);
+        log.info(proxyConfig.logProxyConfig());
```

Comment:
logProxyConfig will check the configuration and return the appropriate message based on the proxy configuration type (proxy profile or default proxy configuration)